python/tank/bootstrap/installed_configuration.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from .configuration import Configuration
+
+from .. import LogManager
+
+log = LogManager.get_logger(__name__)
+
+
+class InstalledConfiguration(Configuration):
+    """
+    Represents a Toolkit pipeline configuration that is installed at a specific location via the
+    ``mac_path``, ``linux_path`` and ``windows_path`` fields.
```

Comment:
maybe worth adding 'that has been setup using the classic setup wizard or tank setup_project command..' or something like that.